pkg/kube/nsmounter

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
```

Comment:
We should put a comment here explaining the purpose of this script.

Reply:
addressed in broken up pr